build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -22,47 +21,47 @@
       <HostOS>Linux</HostOS>
       <DestDir>build-tools\$(XABuildToolsFolder)</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="platform-tools_r28.0.0-linux">
+    <AndroidSdkItem Include="platform-tools_r$(XAPlatformToolsVersion)-linux">
```

Comment:
A concern with this kind of change is a variation on 
https://github.com/xamarin/xamarin-android/pull/2592#issuecomment-454507193.

We have a "mono bundle", and the mono bundle is *invalidated* whenever:

 1. The mono commit hash changes; or
 2. The libzip commit hash changes; or
 3. the llvm commit hash changes; or
 4. The *file content hash* of all files within the `@(VersionFile)` item group within `build-tools/create-bundle/bundle-path.targets` changes.

Ideally, we want to invalidate the mono bundle as infrequently as possible.

`Configuration.props` is *not* in `@(VersionFile)`.  (Nor do we want it in `@(VersionFile)`, as `Configuration.props` can be expected to change more frequently.)

`Ndk.targets` *is* in `@(VersionFile)` (yay).

However, now we hit the philosophical questions: how much -- if any! -- of `build-tools/android-toolchain` *should* be considered for mono bundle invalidation?

`$(AndroidNdkVersion)` is in `Ndk.targets`, which is part of `@(VersionFile)`.  That's fine.

`$(XAPlatformToolsVersion)` is in `Configuration.props`, which is *not* part of `@(VersionFile)`.  *Is that fine*?

I *think* it'll be fine?  I don't see anything in `$HOME/android-toolchain/sdk//platform-tools` which should impact the mono bundle, but we do need to ensure we don't make a mistake.

Reply:
I think it's fine. platform-tools are not used in building Mono.